### DISCUSSION TITLE
License header in each file

### DISCUSSION THREAD

**Discussion by wadoon** [KeYProject/key#3212] [General]
We already discussed the topic of header files. I found in an old branch that we had used 

```java
/*
 * This file is part of KeY - https://key-project.org
 * KeY is licensed by the GNU General Public License Version 2
 * SPDX-License-Identifier: GPL-2.0
 */
```

**Comment by FliegendeWurst** [comment]
I think `GPL-2.0` should be ` GPL-2.0-or-later`.

See
https://github.com/KeYProject/key/blob/0ac6d7f2f89621db5bcd223c76470fdcc857ac4d/LICENSE.TXT#L38-L39

**Reply by wadoon** [reply]
This is not possible. KeY was created under GPLv2-only. We do not have the rights to alter the license.
